generate_golden.sh

```diff
@@ -37,69 +37,46 @@ scaffold_test_project() {
     cd testdata/$project
     kb=$testdata_dir/../bin/kubebuilder
     oldgopath=$GOPATH
-    if [ $version == "1" ]; then
```

Comment:
What do you think about letting this check but just logging a message saying that v1 tests are not supported?

Reply:
What would be the goal of it since it is just used by contributors?

Reply:
If someone adds a v1 test it will be notified instead of passed silently.

Reply:
What you are suggestion is: 

> if [ $version == "1" ]; then
> // do nothing

Why, how someone would add a V1 test since it is deprecated and should no longer have implementations for? And then, if a need as this one started to be required then the if should be added, however, for me make no sense keep an if without need at the moment.

Reply:
What I suggest is to write a message error so that in the future we don't write `
scaffold_test_project project-v2 1` by mistake.

Reply:
Sorry, but I still not thinking that it is required and I do not agree with your suggestion. So, I will let it as it is and if others think that it is required as you then I can follow up the suggestion.